(Updated) Add Granblue Fantasy Wiki presence

kyrie25 opened the pull request:
A presence for https://gbf.wiki/
### Updated from the previous pull request, these presences are available while undergoing these tasks:
  * Logging in  
  * Creating an account    
  * Making a collection tracker  
  * Searching for:  
  * Editing preferences  
  * Viewing watchlist  
  * Looking up contributions  
  * Viewing revision history  
  * Editing  
  * Viewing main page
  * Viewing pages
### Screenshots:
![image](https://user-images.githubusercontent.com/77577746/144690886-d2656ab1-aa90-4976-975d-08b334371015.png)
![image](https://user-images.githubusercontent.com/77577746/144690908-bb90265c-b58b-4f2e-984f-6f985d9e900f.png)
![image](https://user-images.githubusercontent.com/77577746/144690937-cc8c09a9-5b79-4791-9b7e-f6e18be2c0fd.png)
![image](https://user-images.githubusercontent.com/77577746/144690961-19b315f5-dd8b-480e-a052-6bad3c502fb5.png)
![image](https://user-images.githubusercontent.com/77577746/144691062-0b9a3436-23f4-4f38-9443-c96b5470a4af.png)
![image](https://user-images.githubusercontent.com/77577746/144691098-f679d797-29f1-4fd0-9a3d-bba1c6a5dfa6.png)
![image](https://user-images.githubusercontent.com/77577746/144691113-ca1e4302-79cd-4418-8107-11ced455b7d7.png)
![image](https://user-images.githubusercontent.com/77577746/144691259-1115329b-ba54-4f2d-a5a2-c6e839a43650.png)

kyrie25 committed:
Add files via upload

kyrie25 committed:
Update metadata.json

kyrie25 committed:
Merge branch 'PreMiD:main' into add-granblue-fantasy-wiki-presence

kyrie25 committed:
Add files via upload

kyrie25 committed:
Add files via upload

kyrie25 committed:
Update presence.ts

kyrie25 committed:
Update presence.ts

EncryptedDev commented on websites/G/Granblue Fantasy Wiki/presence.ts:
```diff
@@ -0,0 +1,49 @@
+const presence = new Presence({
+    clientId: "914354609370329098"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
```
```suggestion
  browsingTimestamp = Math.floor(Date.now() / 1000);

```

EncryptedDev commented on websites/G/Granblue Fantasy Wiki/presence.ts:
```diff
@@ -0,0 +1,49 @@
+  const presenceData: PresenceData = {
+    largeImageKey: "vyrnball"
+  };
+  presenceData.startTimestamp = browsingStamp;
```
Set this in the PresenceData object

kyrie25 replied:
Fixed

kyrie25 committed:
Update websites/G/Granblue Fantasy Wiki/presence.ts

Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

kyrie25 committed:
Update presence.ts

EncryptedDev commented on websites/G/Granblue Fantasy Wiki/presence.ts:
```diff
@@ -0,0 +1,50 @@
+  const presenceData: PresenceData = {
+    largeImageKey: "vyrnball"
+  };
+  presenceData.startTimestamp = browsingTimestamp;
```
Move this into the PresenceData object above

kyrie25 replied:
Fixed

EncryptedDev commented on websites/G/Granblue Fantasy Wiki/presence.ts:
```diff
@@ -0,0 +1,50 @@
+    presenceData.details = "Editing:";
+    presenceData.state = document.querySelector(".firstHeading").textContent;
+  }
+  else if (document.querySelector(".firstHeading") !== null) {
```
```suggestion
  else if (document.querySelector(".firstHeading")) {
```

kyrie25 committed:
Update presence.ts

kyrie25 committed:
Update websites/G/Granblue Fantasy Wiki/presence.ts

Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

kyrie25 committed:
Added 1 new presence & several small image data

kyrie25 commented:
Minor update:
- Added presence for: `Viewing the character tier list`
- Added small image for: `Viewing the character tier list`, `Logging in`, `Creating an account`, `Searching for:`, `Editing`, `Viewing page:`
![image](https://user-images.githubusercontent.com/77577746/144774596-ccf6d06c-eda7-49ee-bbe7-5583e9ab405f.png)
![image](https://user-images.githubusercontent.com/77577746/144774836-8e312347-247b-43a0-9145-eabf31069b7f.png)

Slowlife01 committed:
style(Granblue Fantasy Wiki)

Slowlife01 commented on websites/G/Granblue Fantasy Wiki/dist/metadata.json:
```diff
@@ -0,0 +1,23 @@
+    "gbf",
+    "wiki"
+  ],
+  "category": "anime"
```
Hmm anime?

kyrie25 replied:
Considering I based this of from Bas950's _Azur Lane Wiki_ and he wrote it down as "anime",and that both games shares the same genre.  
So yeah, anime

kyrie25 replied:
@Slowlife01 what category do you think is better for this presence and I'll fix it ASAP

kyrie25 replied:
Changed it to `game`, minor mistake sorry

kyrie25 committed:
Update metadata.json
